client/src/assets/locales/nb/translation.json

```
@@ -96,11 +96,12 @@
   "oppslag.startPåNytt": "Start på nytt",
 
   "del.lagerstatus.ikkeFastLagervare": "Delen kan ikke bestilles digitalt. Ta kontakt med HMS {{hmsNavn}} for å bestille.",
-  "del.antallDagerSidenSistBatteribestilling_zero": "Det er allerede bestilt batteri i dag. Ta kontakt med Hjelpemiddelsentralen hvis det likevel er behov for nytt batteri.",
-  "del.antallDagerSidenSistBatteribestilling_other": "Det er bestilt batteri for {{count}} dager siden. Ta kontakt med Hjelpemiddelsentralen hvis det likevel er behov for nytt batteri.",
+  
+  "del.antallDagerSidenSistBatteribestilling": "Batteriene til dette hjelpemiddelet ble byttet for mindre enn et år siden, og er innenfor garantitid. Det er derfor ikke mulig å bestille nye batterier via denne tjenesten. Ta kontakt med Nav hjelpemiddelsentral for å avklare om feilen dekkes av garantien.",
+  "del.hjelpemiddelInnenforGaranti": "Hjelpemiddelet er innenfor garantitid. Det er derfor ikke mulig å bestille nye batterier via denne tjenesten. Ta kontakt med Nav hjelpemiddelsentral for å avklare om feilen dekkes av garantien",
```

Mangler et punktum på slutten her. 
```suggestion
  "del.hjelpemiddelInnenforGaranti": "Hjelpemiddelet er innenfor garantitid. Det er derfor ikke mulig å bestille nye batterier via denne tjenesten. Ta kontakt med Nav hjelpemiddelsentral for å avklare om feilen dekkes av garantien.",
```